docs/Getting-Started-Guides/On-Premises-FPGA-Getting-Started/Frequently-Asked-Questions/Frequently-Asked-Questions-Template.rst

```diff
@@ -0,0 +1,28 @@
+Getting Started with On Premises FPGA Frequently Asked Questions
+============================================================================================
+
+To streamline the set-up process, we provide fix suggestions to the following errors that might appear during the process for reference.
+
+**SSH Setup**
+  ``build bitstream`` process stuck at ::
+  
+  [localhost] local: pwd
+  [localhost] run: ...
+  
+  
+  The manager machine should be able to use ssh protocal to connect to build_farm and run_farm machines without providing password or passphrase duirng the connection process. 
```

Comment:
Instead, I would say that there is no passwordless access to the machine in question (i.e. localhost). And I would add a link to SSH setup docs that already exist.